zcash_client_sqlite/src/wallet/init/migrations/add_account_birthdays.rs

```diff
@@ -0,0 +1,75 @@
+//! This migration adds a birthday height to each account record.
+
+use std::collections::HashSet;
+
+use schemer_rusqlite::RusqliteMigration;
+use uuid::Uuid;
+use zcash_primitives::consensus::{self, NetworkUpgrade};
+
+use crate::wallet::init::WalletMigrationError;
+
+use super::shardtree_support;
+
+pub(super) const MIGRATION_ID: Uuid = Uuid::from_u128(0xeeec0d0d_fee0_4231_8c68_5f3a7c7c2245);
+
+pub(super) struct Migration<P> {
+    pub(super) params: P,
+}
+
+impl<P> schemer::Migration for Migration<P> {
+    fn id(&self) -> Uuid {
+        MIGRATION_ID
+    }
+
+    fn dependencies(&self) -> HashSet<Uuid> {
+        [shardtree_support::MIGRATION_ID].into_iter().collect()
+    }
+
+    fn description(&self) -> &'static str {
+        "Adds a birthday height for each account."
+    }
+}
+
+impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
+    type Error = WalletMigrationError;
+
+    fn up(&self, transaction: &rusqlite::Transaction) -> Result<(), Self::Error> {
+        transaction.execute_batch(&format!(
+            "ALTER TABLE accounts ADD COLUMN birthday_height INTEGER;
+                ALTER TABLE accounts ADD COLUMN recover_until_height INTEGER;
```

Comment:
Is this `ALTER TABLE` to add `recover_until_height` necessary? We don't migrate any data into it, and we don't insert anything from it into `accounts_new` below, so I think we can skip this and just create it directly in `accounts_new`.